app/helpers/mailer_helper.rb

```diff
@@ -133,21 +133,36 @@ def creation_title_with_word_count(creation)
       creation_title: title, word_count: creation_word_count(creation))
   end
 
-  # The bylines used in subscription emails to prevent exposing the name(s) of
-  # anonymous creator(s).
-  def creator_links(work)
-    if work.anonymous?
-      "Anonymous"
+  # TODO: Update all mailers to use creator_links or creator_text as
+  # appropriate to eliminate pseuds.map { etc }.to_sentence in the views. Note
+  # that these should never be used on anonymous creations. Text dealing with
+  # Anonymous creations should always have a separate translation, as in the
+  # creation_attribution_ and batch_subscription_subject methods below.
+  def creator_links(creation)
+    creation.pseuds.map { |p| style_pseud_link(p) }.to_sentence.html_safe
```

Comment:
If you use the `ActionView::Helpers::OutputSafetyHelper` version of `to_sentence` here, you get the `html_safe` for free:
```suggestion
    to_sentence(creation.pseuds.map { |p| style_pseud_link(p) })
```
(Side note: I think it'd be good to re-enable the `Rails/OutputSafety` cop, so that people are more aware of when they're calling an unsafe function like `html_safe`, and will hopefully look for alternatives. Ultimately, if you want to fix the big parser issues without introducing XSS exploits, someone is going to need to examine every single call to `html_safe` to make sure that it's actually safe once things aren't being sanitized on input, so the fewer of them there are, the better.)